timbot.py

```diff
@@ -149,4 +162,30 @@ def run_timbot():
 
 
 if __name__ == '__main__':
-    main()
+
+    # load configuration data
+    try:
+        with open('config.yaml', 'r') as file:
+            config = yaml.load(file)
+    except Exception as e:
+        print("Error loading configuration data: ", e)
+        sys.exit()
+
+    # connect to database
+    try:
+        conn = mysql.connector.connect(user=config['mysql']['user'],
+                                       password=config['mysql']['password'],
+                                       host=config['mysql']['host'],
+                                       database=config['mysql']['database'])
```

Comment:
Can we make a variable for the `config['mysql']` and just reference that dictionary when pulling out the database info, e.g. `user=config['user']`

Reply:
Nah